### ISSUE TITLE
Adding Voice Input on Ask AI

### ISSUE THREAD

**Issue by ankurunziza2015-bit** [unicef/adt-studio#680]
### Preliminary Checks

- [x] I have searched existing issues and avoided creating duplicates.

### Where is the issue?

ADT Studio app (the editor interface)

### Description of the barrier

To make the ADT studio more accessible, adding the feature of voice inputs/shots in Ask AI will enable PwD enhance Accessibility 

### Steps to reproduce

Adding voice input will improve the accessibility of ADT Studio.

### Version

v0.7.5-beta.1

**Comment by elasticsounds** [comment]
Yes this has been a long requested feature - we have a demo with Voice input here - it hasn't been standardized. The requirement for adding voice input as an option to interact with the ADT is primarily to do with accuracy in multi-lingual model detection for voice to answer. 

Can you flesh out this specification specifically for what you would want the voice input to do - ie. answer with numbers only to select quiz multiple choice - voice dictation - etc. How would it work exactly and how would we make it reliable? 

Currently I am concerned about reliability in the voice input arena vs. using a keyboard - can we ask the disability experts in Tanzania to weigh in on how this should be implemented?

@ankurunziza2015-bit Can you move this to a github Discussion?
